spec/rspec/core/configuration_spec.rb

```diff
@@ -1229,7 +1229,174 @@ def metadata_hash(*args)
       end
     end
 
+    describe "color options" do
+      context "#force_color=false, #no_color=false" do
+        before { config.force_color = false }
+        before { config.no_color = false }
```

Comment:
There's a bit of overhead for every `before` hook you defined, and I don't really see an advantage to making these separate hooks instead of combining them into one multi-line hook:

``` ruby
before do
  config.force_color = false
  config.no_color = false
end
```

Mind combining them (here and elsewhere in this PR)?